moviepy/video/fx/crop.py

```diff
@@ -32,32 +32,26 @@ def crop(clip, x1=None, y1=None, x2=None, y2=None,
     """
     
     
-    if width and (x1 is not None or x2 is not None):
-        if x1 is not None:
-            x2 = x1+width
-        else:
-            x1 = x2-width
-    
-    if height and (y1 is not None or y2 is not None):
-        if y1 is not None:
-            y2 = y1+height
-        else:
-            y1 = y2 - height
```

Comment:
Similar problems here:
If you pass in `x1=0, y1=0, width=5, height=5` then your version performs incorrectly because all 4 of the above expressions evaluate to `False`. Therefore, `x2` and `y2` get set incorrectly.

Use

```
if width and x1 is not None:
    x2 = x1 + width
```
etc